CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 
 First install:
 
-* Node.js ([v18.17.0 LTS](https://nodejs.org/de/blog/release/v18.17.0) is recommended).
+* Node.js ([v18.17.0 LTS](https://nodejs.org/en/blog/release/v18.17.0) is recommended).
```

Comment:
This is unrelated, but I guess unintentional, and confused me for a little while as my node version was out of date

Reply:
Oops...